Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml

```diff
@@ -119,7 +119,7 @@
       - NotekeeperCartridge
       - NewsReaderCartridge
       - CrimeAssistCartridge
-      
```

Comment:
sus

Reply:
WHITESPACE MUST DIE

Reply:
actually shuld i actually put the spaces back. i remember why that line got fucked its bc i added the mailmetrics line to clerks by accident and then had to move it